tests/acceptance/partial-move-test.js

```diff
@@ -28,94 +29,95 @@ module('Acceptance: Partial move', {
         items: [item.toJSON({includeId: true})]
         }
     });
+
+    $.mockjax({url: '/api/v1/item*', type: 'PUT', status: 200,responseText: {
+        items: [item.toJSON({includeId: true})]
+        }
+    });
+    mockFindAll('location').returns({json: {locations: [lctn.toJSON({includeId: true})]}});
+    mockFindAll('item').returns({ json: {items: [item.toJSON({includeId: true})], orders_packages: [orders_package.toJSON({includeId: true})], meta: {search: item.get('inventoryNumber').toString()}}});
+
+    $.mockjax({url: '/api/v1/stockit_item*', type: 'GET', status: 200,responseText: {
+          items: [item.toJSON({includeId: true})]
+        }
+    });
+
+    //visiting Item's partial move
+    visit("/items/" + item.id + "/partial_move");
+
```

Comment:
so we want to check there is expected output on the page here? We are going to the partial_move page but not checking for anything other than clicking the move link